core/micrometer/src/test/java/MicrometerExtensionIntegrationTest.java

```diff
@@ -0,0 +1,66 @@
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+import org.eclipse.dataspaceconnector.azure.testfixtures.annotations.OpenTelemetryIntegrationTest;
+import org.eclipse.dataspaceconnector.common.annotations.IntegrationTest;
+import org.eclipse.dataspaceconnector.junit.launcher.EdcExtension;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import java.io.IOException;
+import java.lang.management.ManagementFactory;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.eclipse.dataspaceconnector.common.testfixtures.TestUtils.getFreePort;
+
+@IntegrationTest
+@OpenTelemetryIntegrationTest
+@ExtendWith(EdcExtension.class)
+public class MicrometerExtensionIntegrationTest {
+    static final int CONNECTOR_PORT = getFreePort();
+    static final String CONNECTOR_URL = String.format("http://localhost:%s", CONNECTOR_PORT);
+    static final String HEALTH_ENDPOINT = String.format("%s/api/check/health", CONNECTOR_URL);
```

Comment:
nit: both constants can be merged and replaced by `HEALTH_ENDPOINT` only. Just to reduce extra constant because seems like `CONNECTOR_URL` is not used in the test code.